scripts/cord19/find_cord19_length_outlier.py

```diff
@@ -19,7 +19,7 @@
 
```

Comment:
Maybe add a description of what this script does?

Reply:
Yup, good call. This script was written by an URA. I'll ask for improvement on next iteration.